nav2_bringup/launch/nav2_bringup_launch.py

```diff
@@ -4,16 +4,13 @@
 import launch_ros.actions
 
 def generate_launch_description():
-    map_file = launch.substitutions.LaunchConfiguration('map')
-    map_type = launch.substitutions.LaunchConfiguration('map_type', default='occupancy')
+    map_yaml_file = launch.substitutions.LaunchConfiguration('map')
```

Comment:
I just want to clarify, in the launch file we still pass the map file name, not the params file name, right?

Reply:
@mkhansen-intel Yes, we can continue to pass the map file name to the launch script. In the launch script this filename is set as a parameter to the map server node (behind the scenes, the launch script creates a temporary YAML file which contains this parameter).